core/src/main/java/de/danoeh/antennapod/core/sync/SyncService.java

```diff
@@ -59,6 +61,13 @@
 import java.util.concurrent.locks.ReentrantLock;
 
 public class SyncService extends Worker {
+    public static final String SHARED_PREFERENCES_SYNCHRONIZATION = "synchronization";
+    public static final String SHARED_PREFERENCE_SELECTED_SYNC_PROVIDER = "selected_sync_provider";
+    public static final String SHARED_PREFERENCE_IS_SYNC_PROVIDER_CONNECTED = "provider_is_connected";
+    public static final String SYNC_PROVIDER_CHOICE_GPODDER_NET = "GPodder.net";
+    public static final String SYNC_PROVIDER_CHOICE_NEXTCLOUD = "Nextcloud";
```

Comment:
I think it could lead to confusion when the choice `keys` are the actual text that is displayed to users. I think it would be better to introduce internal keys (with a format that clearly shows that it is not a user-visible text, for example `nextcloud_gpodder`) for storage and only use the full name in the UI. Otherwise, when adding the other nextcloud sync provider (apparently, there are even 3 different ones now), the stored values would need to be migrated when doing UI changes

Reply:
They are not displayed to users.

Reply:
I would argue that these internal names should be attributes of the sync provider enum. Adding a new sync provider then needs to touch fewer places.